web-access-control/web-access-control-test-manifest.ttl

```diff
@@ -66,55 +66,55 @@ manifest:acl-object-default
 manifest:acl-object-access-to-default
   a td:TestCase ;
   spec:requirementReference <https://solid.github.io/web-access-control-spec#access-objects> ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   td:preCondition "authentication", "acl" ;
   spec:testScript
     <https://github.com/solid/specification-tests/web-access-control/acl-object/container-access-to-default.feature> .
 
 manifest:protected-operation-acl-propagation
   a td:TestCase ;
   spec:requirementReference <https://solid.github.io/web-access-control-spec#authorization-evaluation-context> ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   td:preCondition "authentication", "acl" ;
   spec:testScript
     <https://github.com/solid/specification-tests/web-access-control/protected-operation/acl-propagation.feature> .
 
 manifest:server-wac-allow-header-exists
   a td:TestCase ;
   spec:requirementReference <https://solid.github.io/web-access-control-spec#server-wac-allow> ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   td:preCondition "authentication", "acl";
   spec:testScript
     <https://github.com/solid/specification-tests/web-access-control/wac-allow/header-exists.feature> .
 
 manifest:server-wac-allow-user-access-direct
   a td:TestCase ;
   spec:requirementReference <https://solid.github.io/web-access-control-spec#server-wac-allow> ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   td:preCondition "authentication", "acl";
   spec:testScript
     <https://github.com/solid/specification-tests/web-access-control/wac-allow/user-access-direct.feature> .
```

Comment:
The `only` was not clear to me (but eventually became clear after studying the code; just not ideal)

Reply:
I don't understand several of the scenario titles, for example:
- There is an acl on the resource containing Bob's WebID
- There is no acl on the parent that references Bob
- Alice calls GET and the header shows full access for user

Wordings like "call GET" are strange to me.

Reply:
We'll try and make the `only` setting clearer - this is the first time we used table driven tests.

The initial acl tests were to confirm that there were no other ACLs influencing the test but they are possibly too naive anyway. We'll look at title wording and apply a style across all tests.